stable/cockroachdb/README.md

```diff
@@ -101,6 +101,7 @@ The following table lists the configurable parameters of the CockroachDB chart a
 | `Secure.ServiceAccount.Create` | Whether to create a new RBAC service account     | `true`                                    |
 | `Secure.ServiceAccount.Name`   | Name of RBAC service account to use              | `""`                                      |
 | `JoinExisting`                 | List of already-existing cockroach instances     | `[]`                                      |
+| `JoinReplicas`                 | number of replicas to join, not used if `JoinExisting` is specified  | `3`                   |
```

Comment:
Do you actually want a custom number of `JoinReplicas`? Or is the change to only specify join flags if the number of replicas is greater than 1 good enough? My intuition would be that this new parameter is unlikely to be used, and that your new conditional is the right fix.